packages/grid/_modules_/grid/hooks/features/sorting/useGridSorting.ts

```diff
@@ -3,7 +3,6 @@ import {
   GRID_COLUMN_HEADER_CLICK,
   GRID_COLUMN_HEADER_KEYDOWN,
   GRID_COLUMNS_UPDATED,
-  GRID_ROWS_CLEARED,
```

Comment:
you can remove the constant as well and update the api

Reply:
This would be a breaking change. I think that we can decorate the two. My objective here is to simplify the logic.

Reply:
we are in alpha. Let's do it I would say